setup.py

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env python3
 
 """
-Custom setup.py to install some, or all, mewbot pckages.
-
```

Comment:
I think changes in this file all want to be reverted.

Reply:
Agreed. Corrected.